ApplicationLibCode/UnitTests/RiaStdStringTools-Test.cpp

```diff
@@ -202,3 +202,31 @@ TEST( RiaStdStringToolsTest, DISABLED_PerformanceConversion )
         std::cout << "std::from_chars " << std::setw( 9 ) << diff.count() << " s\n";
     }
 }
+
+//--------------------------------------------------------------------------------------------------
+///
+//--------------------------------------------------------------------------------------------------
+TEST( RiaStdStringToolsTest, ValuesFromRangeSelection )
+{
+    std::string   testString     = "1,2,5-10,15";
+    std::set<int> expectedValues = { 1, 2, 5, 6, 7, 8, 9, 10, 15 };
+
+    auto actualValues = RiaStdStringTools::valuesFromRangeSelection( testString );
+
+    ASSERT_EQ( expectedValues, actualValues );
+}
+
+//--------------------------------------------------------------------------------------------------
+///
+//--------------------------------------------------------------------------------------------------
+TEST( RiaStdStringToolsTest, ValuesFromRangeSelectionMinMax )
+{
+    std::string   testString     = "-3, 5, 6-8, 10, 15-";
+    int           minimumValue   = 1;
```

Comment:
Maybe add some tests for misconfigured queries. `"4-3, -99"`, ` "-1,-3"`

Reply:
Thanks, added more tests and improved code.